lib/model/query/blobs.js

```diff
@@ -31,5 +31,17 @@ const getById = (blobId) => ({ maybeOne }) =>
   maybeOne(sql`select * from blobs where id=${blobId}`)
     .then(map(construct(Blob)));
 
-module.exports = { ensure, getById };
+const purgeUnattached = () => ({ all }) =>
+  all(sql`
+delete from blobs
```

Comment:
I was wondering, for this purging of unattached blobs and for deleting forms and submissions over all, would it make sense to delete things in batches? Like, keep deleting blobs/submissions/etc. N-at-a-time until there are no more to delete? (Leaving this question here to ask about later...)